ERROR - 2016-07-19 07:23:40 - exception 'ErrorException' with message 'Undefined variable: error' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:57
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(57): Yee\Yee::handleErrors(8, 'Undefined varia...', 'C:\\wamp64\\www\\A...', 57, Array)
#1 [internal function]: AjaxController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: AjaxController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 07:23:42 - exception 'ErrorException' with message 'Undefined variable: error' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:57
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(57): Yee\Yee::handleErrors(8, 'Undefined varia...', 'C:\\wamp64\\www\\A...', 57, Array)
#1 [internal function]: AjaxController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: AjaxController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 07:23:43 - exception 'ErrorException' with message 'Undefined variable: error' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:57
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(57): Yee\Yee::handleErrors(8, 'Undefined varia...', 'C:\\wamp64\\www\\A...', 57, Array)
#1 [internal function]: AjaxController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: AjaxController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 07:27:36 - exception 'ErrorException' with message 'Undefined index: loginEmail' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:30
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(30): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 30, Array)
#1 [internal function]: AjaxController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: AjaxController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 07:29:00 - exception 'ErrorException' with message 'Undefined variable: email' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:35
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(35): Yee\Yee::handleErrors(8, 'Undefined varia...', 'C:\\wamp64\\www\\A...', 35, Array)
#1 [internal function]: AjaxController->loginAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___loginAction', Array)
#4 [internal function]: AjaxController::___loginAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 07:30:07 - exception 'ErrorException' with message 'Undefined variable: email' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:35
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(35): Yee\Yee::handleErrors(8, 'Undefined varia...', 'C:\\wamp64\\www\\A...', 35, Array)
#1 [internal function]: AjaxController->loginAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___loginAction', Array)
#4 [internal function]: AjaxController::___loginAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 07:30:28 - exception 'ErrorException' with message 'Undefined variable: email' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:35
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(35): Yee\Yee::handleErrors(8, 'Undefined varia...', 'C:\\wamp64\\www\\A...', 35, Array)
#1 [internal function]: AjaxController->loginAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___loginAction', Array)
#4 [internal function]: AjaxController::___loginAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 07:31:22 - exception 'ErrorException' with message 'Undefined variable: email' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:35
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(35): Yee\Yee::handleErrors(8, 'Undefined varia...', 'C:\\wamp64\\www\\A...', 35, Array)
#1 [internal function]: AjaxController->loginAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___loginAction', Array)
#4 [internal function]: AjaxController::___loginAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 07:35:35 - exception 'ErrorException' with message 'Undefined variable: email' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:35
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(35): Yee\Yee::handleErrors(8, 'Undefined varia...', 'C:\\wamp64\\www\\A...', 35, Array)
#1 [internal function]: AjaxController->loginAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___loginAction', Array)
#4 [internal function]: AjaxController::___loginAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 08:25:29 - exception 'ErrorException' with message 'Use of undefined constant Yee - assumed 'Yee'' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:18
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(18): Yee\Yee::handleErrors(8, 'Use of undefine...', 'C:\\wamp64\\www\\A...', 18, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 08:25:41 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(23): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:25:53 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(23): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:13 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:18 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:25 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:25 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:26 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:26 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:28 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:46 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:54 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:34:55 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:38:47 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:38:53 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:38:53 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:39:02 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:39:09 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:39:17 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:39:21 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:40:26 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:40:46 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:40:46 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:40:55 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:41:36 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:42:30 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:43:54 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:44:03 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:44:09 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:44:17 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:44:25 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:44:33 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:44:50 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:45:50 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:46:13 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:49:49 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:49:59 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:50:44 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username = ?  LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:50:47 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username = ?  LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:50:48 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username = ?  LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:51:04 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username = ?  LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:51:05 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username = ?  LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:51:06 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username = ?  LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:51:06 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username = ?  LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:52:48 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username = ?  LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:53:01 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:53:02 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:53:02 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:53:08 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 08:55:36 - exception 'ErrorException' with message 'Problem preparing query (SELECT * FROM users WHERE   username = ?  LIMIT 1) Unknown column 'username' in 'where clause'' in C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\wamp64\\www\\Y...', 909, Array)
#1 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(273): Yee\Libraries\Database\MysqliDB->_buildQuery(1)
#4 C:\wamp64\www\Yee\Libraries\Database\MysqliDB.php(297): Yee\Libraries\Database\MysqliDB->get('users', 1, '*')
#5 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(22): Yee\Libraries\Database\MysqliDB->getOne('users')
#6 C:\wamp64\www\App\Models\AjaxLogin\AjaxLoginModel.php(60): App\Models\AjaxLogin\AjaxLoginModel->Ajax_emailexist()
#7 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(24): App\Models\AjaxLogin\AjaxLoginModel->validate()
#8 [internal function]: AjaxController->ajaxLogin()
#9 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#10 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#11 [internal function]: AjaxController::___ajaxLogin()
#12 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#13 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#14 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#15 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#16 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#17 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#18 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#19 {main}
ERROR - 2016-07-19 09:52:10 - exception 'ErrorException' with message 'Array to string conversion' in C:\wamp64\www\App\Controllers\Home\HomeController.php:35
Stack trace:
#0 C:\wamp64\www\App\Controllers\Home\HomeController.php(35): Yee\Yee::handleErrors(8, 'Array to string...', 'C:\\wamp64\\www\\A...', 35, Array)
#1 [internal function]: HomeController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: HomeController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('HomeController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 09:52:46 - exception 'ErrorException' with message 'Array to string conversion' in C:\wamp64\www\App\Controllers\Home\HomeController.php:30
Stack trace:
#0 C:\wamp64\www\App\Controllers\Home\HomeController.php(30): Yee\Yee::handleErrors(8, 'Array to string...', 'C:\\wamp64\\www\\A...', 30, Array)
#1 [internal function]: HomeController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: HomeController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('HomeController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 09:52:48 - exception 'ErrorException' with message 'Array to string conversion' in C:\wamp64\www\App\Controllers\Home\HomeController.php:30
Stack trace:
#0 C:\wamp64\www\App\Controllers\Home\HomeController.php(30): Yee\Yee::handleErrors(8, 'Array to string...', 'C:\\wamp64\\www\\A...', 30, Array)
#1 [internal function]: HomeController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: HomeController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('HomeController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 09:52:48 - exception 'ErrorException' with message 'Array to string conversion' in C:\wamp64\www\App\Controllers\Home\HomeController.php:30
Stack trace:
#0 C:\wamp64\www\App\Controllers\Home\HomeController.php(30): Yee\Yee::handleErrors(8, 'Array to string...', 'C:\\wamp64\\www\\A...', 30, Array)
#1 [internal function]: HomeController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: HomeController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('HomeController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 09:52:48 - exception 'ErrorException' with message 'Array to string conversion' in C:\wamp64\www\App\Controllers\Home\HomeController.php:30
Stack trace:
#0 C:\wamp64\www\App\Controllers\Home\HomeController.php(30): Yee\Yee::handleErrors(8, 'Array to string...', 'C:\\wamp64\\www\\A...', 30, Array)
#1 [internal function]: HomeController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: HomeController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('HomeController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 09:52:52 - exception 'ErrorException' with message 'Array to string conversion' in C:\wamp64\www\App\Controllers\Home\HomeController.php:30
Stack trace:
#0 C:\wamp64\www\App\Controllers\Home\HomeController.php(30): Yee\Yee::handleErrors(8, 'Array to string...', 'C:\\wamp64\\www\\A...', 30, Array)
#1 [internal function]: HomeController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: HomeController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('HomeController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 10:00:57 - exception 'ErrorException' with message 'Array to string conversion' in C:\wamp64\www\App\Controllers\Home\HomeController.php:30
Stack trace:
#0 C:\wamp64\www\App\Controllers\Home\HomeController.php(30): Yee\Yee::handleErrors(8, 'Array to string...', 'C:\\wamp64\\www\\A...', 30, Array)
#1 [internal function]: HomeController->indexAction()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#4 [internal function]: HomeController::___indexAction()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('HomeController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:26:21 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:26:22 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:26:53 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:26:53 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:27:02 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:27:02 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:27:11 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:27:11 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:27:14 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
ERROR - 2016-07-19 12:27:16 - exception 'ErrorException' with message 'Undefined index: http://localhost/home' in C:\wamp64\www\App\Controllers\Ajax\AjaxController.php:45
Stack trace:
#0 C:\wamp64\www\App\Controllers\Ajax\AjaxController.php(45): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\wamp64\\www\\A...', 45, Array)
#1 [internal function]: AjaxController->ajaxLogin()
#2 C:\wamp64\www\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#3 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___ajaxLogin', Array)
#4 [internal function]: AjaxController::___ajaxLogin()
#5 C:\wamp64\www\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#6 C:\wamp64\www\Yee\Yee.php(1334): Yee\Route->dispatch()
#7 C:\wamp64\www\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#8 C:\wamp64\www\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#9 C:\wamp64\www\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#10 C:\wamp64\www\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#11 C:\wamp64\www\index.php(48): Yee\Yee->execute()
#12 {main}
